src/components/EditorSideBar/Blueprints.tsx

```diff
@@ -18,13 +18,13 @@ const Blueprints: React.FC = () => {
     dispatch(changeCourtName(`${courtSizeName} ${courtSizeDetails}`)); // change TopBar court size name
   };
 
-  const { data, error, isLoading, isSuccess, isError } = useGetCourtsQuery();
-  console.log(data);
+  // TODO: use later
+  // const { data, error, isLoading, isSuccess, isError } = useGetCourtsQuery();
 
   return (
     <Box paddingLeft="24px" paddingTop="24px" height="100%" className="scrollbox">
       {courtList.map((court) => {
-        const { img, courtSizeName, courtSizeDetails } = court;
+        const { img, courtSizeName, courtSizeDetails } = court; // better rename to 'imgUrl'
```

Comment:
You can rename them from the source file(../ChangeCourtSize/CourtList) or map them into new names here.